tooltester/src/main/java/io/dockstore/tooltester/jenkins/JenkinsPipeline.java

```diff
@@ -0,0 +1,126 @@
+package io.dockstore.tooltester.jenkins;
+
+/**
+ * @author gluu
+ * @since 08/02/17
+ */
+
+import java.util.List;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class JenkinsPipeline {
```

Comment:
If these are generated classes (they better be), put a @Generated annotation on them. 
Also document in a readme or similar how to re-generate them.